### PR TITLE
Add VoxCeleb dataset class.

### PR DESCRIPTION
**Pull request by bantin**
Add VoxCeleb dataset class and an example notebook fitting a short audio clip.

### CODE REVIEW

**Comment by bantin** [comment]
Requesting review. Currently, during the call to `__init__` we look for the .wav files and concatenate them. Then, during the call to `generate()` we create the spectrogram. My thinking was that a user might want to generate a spectrogram with different parameters (overlap, zero padding, etc) from the same set of audio files.

File: cmfpy/datasets/vox_celeb.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import scipy.signal as signal
+import sklearn.preprocessing as preprocessing
+import warnings
+
+DATAPATH = os.path.expanduser("~/cmf_data/VoxCeleb/")
+
+class VoxCeleb:
+    """Spectrogram data from the Vox Celeb Dataset."""
+
+    def __init__(self, secs, path, concat=True):
```
Comment (degleris1) [inline cmfpy/datasets/vox_celeb.py]: Could we possibly also include the starting time? So someone could load the last five minutes for example.

Comment (bantin) [reply cmfpy/datasets/vox_celeb.py]: We could, it gets a little tricky though because I think in most cases the user will be passing a path to a folder that contains multiple files – so in that case the question becomes: the last five seconds of which file?
